ziti/cmd/ziti/cmd/config_templates/controller.yml

```diff
@@ -136,33 +88,33 @@ web:
     bindPoints:
       #interface - required
       # A host:port string on which network interface to listen on. 0.0.0.0 will listen on all interfaces
-      - interface: {{.Web.BindAddress}}:{{ .Web.Port }}
+      - interface: 0.0.0.0:{{ .ZitiEdgeCtrlPort }}
```

Comment:
the bind address should definitely be configurable if possible. does that add too much effort to accomplish?

Reply:
Not a problem, will fix